resources/models/resource.py

```diff
@@ -262,6 +262,15 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
                                                                       null=True, blank=True)
     reservable_min_days_in_advance = models.PositiveSmallIntegerField(verbose_name=_('Reservable min. days in advance'),
                                                                       null=True, blank=True)
+    cancellation_min_days_in_advance = models.PositiveIntegerField(
+        verbose_name=_('Reservation cancellation min. days in advance'),
```

Comment:
```suggestion
        verbose_name=_('Reservation can be canceled up to x days in advance'),
```
This is not a very good name either as it contains the x, so please think if you could come up with some that is a bit more clear.

Reply:
Changed to what was discussed in Slack